README.md

```diff
@@ -57,6 +57,24 @@ Sets the clipboard method to either of these options:
 | `pb`         | Use `pbcopy` and `pbpaste` on Darwin - method used by default on OSx. |
 | `termux`     | Use [Termux:API](https://wiki.termux.com/wiki/Termux:API) - method used by default on Android's termux |
 
+If you'd like to change the current clipboard method without restarting your shell - you can run this command:
+
+```zsh
+# Needed only for the first time of course. If you don't set XDG_CONFIG_HOME,
+# use simply ~/.config .
+mkdir -p ${XDG_CONFIG_HOME:-$HOME/.config}/zsh-system-clipboard
+# Here _method_ is any method explained above in the first column.
+echo _method_ > ${XDG_CONFIG_HOME:-$HOME/.config}/zsh-system-clipboard/method
+```
+
+Here's an example use-case: You use tmate inside a Windows Subsystem for Linux (WSL), and you access that tmate session remotely, also when the Windows host is locked after you left the station. You may experience after it is locked that `clip.exe` is being denied access to the system clipboard. However, you still want to be able to copy and paste from tmate's buffers to & from Neovim seamlessly with the `+` / `*` buffers. You can do that by:
+
+```zsh
+echo tmate > ${XDG_CONFIG_HOME:-$HOME/.config}/zsh-system-clipboard/method
+```
+
+And in the Neovim session:
```

Comment:
Is this part incomplete, or should we leave it as like this?

Reply:
Aish I just realized why I didn't push this diff for such a long time - it wasn't complete :sweat_smile: . Luckily I wasn't arrogant enough to simply push this incomplete change. Perhaps I'll keep this as a draft until I'll find time to complete this feature.

Reply:
Okay, no worries. 😃